jobs/pull-kubernetes-bazel.sh

```diff
@@ -28,14 +28,16 @@ release-1.0|release-1.1|release-1.2|release-1.3|release-1.4|release-1.5)
   ;;
 esac
 
-# TODO(ixdy): remove once all branches are back to build/
-build_dir=build
-if [[ -d build-tools/ ]]; then
-  build_dir=build-tools
+bazel build //cmd/... //pkg/... //federation/... //plugin/... //build/... //examples/... //test/... //third_party/... && rc=$? || rc=$?
+
+if [[ "${rc}" != 0 ]]; then
```

Comment:
This produces the intended behavior and aligns with the style guide which days to quote all variable expansions unless it's necessary not to